test/functional/test_framework/test_framework.py

```
@@ -263,6 +263,23 @@ def parse_args(self):
 
         PortSeed.n = self.options.port_seed
 
+    def set_binary_paths(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `set_binary_paths`

Add return type annotation: `None`

(ANN201)